### PR TITLE
tests/core/gadget-config-defaults: fix cleanup

### PR DESCRIPTION
**Pull request by valentindavid**
Path for the mount unit name  was not properly escaped.

### CODE REVIEW

File: tests/core/gadget-config-defaults/task.yaml [hunk 1]
```diff
@@ -117,14 +117,11 @@ restore: |
     restore_pc_snap
 
     TEST_REVNO=$(awk "/^snap-revision: / {print \$2}" "test-snapd-with-configure${SUFFIX}"_*.assert)
-    if systemctl status "$(systemd-escape --path "/snap/test-snapd-with-configure${SUFFIX}/$TEST_REVNO.mount")"; then
```
Comment (sergiocazzolato) [inline tests/core/gadget-config-defaults/task.yaml:120]: It is needed to update the test gadget-config-defaults-vitality as well.

Comment (valentindavid) [reply tests/core/gadget-config-defaults/task.yaml:120]: Done.

Comment (sergiocazzolato) [reply tests/core/gadget-config-defaults/task.yaml:120]: tx

**Review by sergiocazzolato — APPROVED** [review]
Awesome, thanks for this fix

**Review by mvo5 — APPROVED** [review]

**Pull request merged**